x-pack/metricbeat/module/sql/query/query.go

```diff
@@ -44,24 +51,36 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	cfgwarn.Beta("The sql query metricset is beta.")
 
 	config := struct {
-		Driver string `config:"driver" validate:"nonzero,required"`
-		Query  string `config:"sql_query" validate:"nonzero,required"`
+		Driver         string `config:"driver" validate:"nonzero,required"`
+		Query          string `config:"sql_query" validate:"nonzero,required"`
+		ResponseFormat string `config:"response_format"`
```

Comment:
I meant to change only the name of the variable, but not the name of the config option.
```suggestion
		ResponseFormat string `config:"sql_response_format"`
```

But as you prefer, both options look good to me.